backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/TeamService.cs

```diff
@@ -124,19 +126,23 @@ public async Task<List<TeamDto>> GetCurrentUserTeams()
             .ToListAsync();
 
         return _mapper.Map<List<TeamDto>>(teams);
-    } 
+    }
     public async Task<ICollection<NewMeetingMemberDto>> GetTeamMembersOfCurrentUserAsync()
```

Comment:
I think we can pass current user team and get team members of currently selected team only?

Reply:
Fixed it.